tests/project.bats

```diff
@@ -27,63 +27,42 @@ teardown() {
 @test "fin project start" {
 	[[ $SKIP == 1 ]] && skip
 
-	run fin project start
-	echo "$output" | egrep "Creating network \".*_default\" with the default driver"
-	echo "$output" | egrep "Creating volume \".*_project_root\" with local driver"
-	echo "$output" | egrep "Creating .*_web_1"
-	echo "$output" | egrep "Creating .*_db_1"
-	echo "$output" | egrep "Creating .*_cli_1"
-
-	echo "$output" | egrep "Connected vhost-proxy to \".*_default\" network"
-	unset output
+	fin project start
 
 	# Check that containers are running
 	run fin ps
-	echo "$output" | grep "web_1" | grep "Up"
-	echo "$output" | grep "db_1" | grep "Up"
-	echo "$output" | grep "cli_1" | grep "Up"
+	echo "$output" | grep "_web_1" | grep "running"
+	echo "$output" | grep "_db_1" | grep "running"
+	echo "$output" | grep "_cli_1" | grep "running"
 	unset output
 }
 
 @test "fin project stop" {
 	[[ $SKIP == 1 ]] && skip
 
-	run fin project stop
-	echo "$output" | egrep "Stopping .*_web_1"
-	echo "$output" | egrep "Stopping .*_db_1"
-	echo "$output" | egrep "Stopping .*_cli_1"
-	unset output
+	fin project stop
```

Comment:
Seems like all the tests from here and below has been castrated, and do not perform any real tests that things are going as they should, but rather just check for the exit code. Just throwing in concerns here that it might not provide adequate testing of fin now.

Reply:
Fair point.
Compose v1 and v2 generate different intermediary outputs. Accounting for both would bloat the tests cases.
If we drop everything around the v1 implementation, then this would not be an issue.

Reply:
Updated `fin project` tests for Docker Compose v2